vxquery-xtest/results/xqts.txt

```diff
@@ -10366,7 +10366,7 @@ Expressions/SeqExpr/FilterExpr//K-FilterExpr-36, EXPECTED_RESULT_GOT_SAME_RESULT
 Expressions/SeqExpr/FilterExpr//K-FilterExpr-37, EXPECTED_RESULT_GOT_ERROR
 Expressions/SeqExpr/FilterExpr//K-FilterExpr-38, EXPECTED_ERROR_GOT_RESULT
 Expressions/SeqExpr/FilterExpr//K-FilterExpr-39, EXPECTED_RESULT_GOT_ERROR
-Expressions/SeqExpr/FilterExpr//K-FilterExpr-4, EXPECTED_ERROR_GOT_SAME_ERROR
+Expressions/SeqExpr/FilterExpr//K-FilterExpr-4, EXPECTED_ERROR_GOT_RESULT
```

Comment:
According to xquery specs a filtered expression consists of a primary expression and zero or more predicates. However, both K-FilterExpr-4 and K-FilterExpr-5 consist only of predicates and not of primary expression. So, this should be an error for XQuery but for jsoniq, this is an array, so it gets a result. 

K-FilterExpr-4 query: [true()]